docs/user-guide/replicasets.md

```diff
@@ -36,6 +36,9 @@ provides declarative updates to pods along with a lot of other useful features.
 Therefore, we recommend using Deployments instead of directly using Replica Sets, unless
 you require custom update orchestration or don't require updates at all.
 
+This actually means that you should never need to write a replica set object:
+write directly a Deployment and define your application in the spec section.
+
 ### Example
```

Comment:
It'd make more sense to create a Deployment and then get / describe the created Replica Set after adding the above sentence. cc @madhusudancs